recipes/fastk/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set name = "FASTK" %}
+{% set version = "1.0" %}
+{% set sha256 = "80d483baa74afe491ed5a9b7444c5600d66c0b35fef82d48e5129aa90fb99e17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/abueg/FASTK/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true [osx]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - bzip2
+    - zlib
+    - libcurl ==7.61.0
+
+test:
+  commands:
+    - 'FastK'
```

as it returns a non-zero exit code, the easier method is to just check if the output contains something that is expected
```suggestion
    - 'FastK | grep "Usage: FastK"'
```